### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SSAHMED007**
I am learning to make MR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,6 @@
 
 A kde plasma accent color changer that changes some colors in current colorscheme.
 
-## Getting Started!
+Please Please accept my MR
```
Comment (tantalising) [inline README.md:5]: YOU added this line. In a real MR developers would have yelled at you for some reason, if it was some real code. That's how most of the reviews are done.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,6 @@
...
 
-This project is a starting point for a Flutter application.
 
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
-
```
Comment (tantalising) [inline README.md:13]: You deleted this line. In a real MR, devs would have questioned why you deleted this even if that is mentioned in the description.

File: README.md [hunk 3]
```diff
@@ -2,15 +2,6 @@
...
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
```
Comment (tantalising) [inline README.md:15]: Just imagine someone else came from somewhere and suggests you should rewrite it in a completely different way and makes you MR difficult to get merged.

**Review by tantalising — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (SSAHMED007) [reply README.md:13]: First get selected in gsoc then talk with me

File: README.md (hunk 1 above)
Comment (SSAHMED007) [reply README.md:5]: Your project has copy pasted code

File: README.md (hunk 3 above)
Comment (SSAHMED007) [reply README.md:15]: Okay.. please accept..i just show you on kde community...and you are quite popular there so i came here at your profile and this project makes me worth coming.. accept it please

File: README.md (hunk 2 above)
Comment (SSAHMED007) [reply README.md:13]: Okay

File: README.md (hunk 1 above)
Comment (SSAHMED007) [reply README.md:5]: Okay

File: README.md (hunk 3 above)
Comment (tantalising) [reply README.md:15]: too much man! you are sitting beside me currently lol

**Pull request merged**